qcodes/tests/drivers/test_ami430.py

```diff
@@ -253,6 +253,8 @@ def get_ramp_down_order(messages: List[str]) -> List[str]:
             continue
 
         g = re.search(r"\[(.*).*\] Writing: CONF:FIELD:TARG", msg)
+        if g is None:
+            raise RuntimeError("No match is found")
```

Comment:
Perhaps more explicit message?
```suggestion
            raise RuntimeError(f"No match found in {msg!r} when getting ramp down order")
```